src/jit/importer.cpp

```diff
@@ -5727,10 +5733,10 @@ GenTreePtr Compiler::impImportStaticFieldAccess(CORINFO_RESOLVED_TOKEN* pResolve
             FieldSeqNode* fs = GetFieldSeqStore()->CreateSingleton(pResolvedToken->hField);
             op1              = gtNewOperNode(GT_ADD, type, op1,
                                 new (this, GT_CNS_INT) GenTreeIntCon(TYP_I_IMPL, pFieldInfo->offset, fs));
+            break;
```

Comment:
If this can now fall-through into the next case then you need to add a fallthrough keyword,
 otherwise you need to move the break back where it was,

  __fallthrough;